### ISSUE TITLE
PacketInboundHandler exception

### ISSUE THREAD

**Issue by Gunner76th** [RS485/LogisticsPipes#976]
http://pastebin.com/JegAn910

I am having this error with FTB Infinity Evolved Skyblock. The setup is a coke oven with a crafting pipe that has the http://rs485.network/wiki/Crafting_Byproduct_Extraction_Upgrade installed, and has an active crafting in process (crafting is for more than 64 pieces of coke, so there is coal waiting to be inserted). When I right click on the pipe with the Logistics Pipe Controller, and I click on the last tab (it has lines and dashes on it with no tooltip explaining its reason for being), the client DC's and the server kicks the above error to the console. Neither the client nor the server actually crashes, so no actual crash report is generated. The client is able to log back on with no issues. 

Steps to reproduce:
Be running a Server and Client setup
Running FTB Infinity Evolved Skyblock version 1.1.0
Have a coke oven with a crafting pipe attached, with the Crafting Byproduct Extraction Upgrade installed
Set crafting pipe to insert 2 coal 1 bottle, with primary output being coke, and byproduct as creosote bottle
Submit a request to the system to produce coke (in my issue it was 128+ units)
While crafting is taking place (purple particles on both the crafting pipe), right click on Crafting Pipe with Logistics Pipe Controller
On the GUI that opens, click the last tab on the right (looks like lines and dashes)
At this point the client should DC from the server, with the message appearing on the "Dirt Screen" of "A fatal error has occured, this connection is terminated" with the above pastbin appearing on the server console.

**Comment by hron84** [comment]
@Gunner76th Did you tried this "click on the last tab" thing on a setup that does not exactly like this? E.g. installing a byproduct upgrade on a furnce or similar? If not, try to do this, and please report how it was going.

**Comment by Gunner76th** [comment]
No I have not done it with other pipes with byproduct upgrades. I did however break the power junction, the chassis that was requesting the large amount of coke, as well as the crafting pipe on the coke over. Upon replacing all of them, and without making a crafting request, I repeated the procedure, and no DC occurred. Because the last tab is a display of all pending craftings (least that is my understanding after looking at other pipes on my network), I believe it was caused by there being more craftings than it knew how to display.
